dispatch/static/manager/src/js/pages/DashboardPage.js

```diff
@@ -1,14 +1,85 @@
 import React from 'react';
 import DocumentTitle from 'react-document-title';
+import { Link } from 'react-router'
+import { connect } from 'react-redux'
+import * as dashboardActions from '../actions/DashboardActions'
+import moment from 'moment'
 
-export default class DashboardPage extends React.Component {
+class DashboardPageComponent extends React.Component {
+
+  componentWillMount() {
+    this.props.fetchActions(this.props.token)
+    this.props.fetchRecent(this.props.token)
+  }
 
   render() {
+
+    //returns the user history if the data is loaded, else returns an empty list element
+    let historyList = this.props.actions.isLoaded
+    ? this.props.actions.data.map((elem,ind) => (
+      <li key={ind}>
+        <i className={`fa fa-${elem.icon}`} aria-hidden="true"></i>
+        {elem.meta.author}
+        {elem.meta.count == 1 ? ` ${elem.meta.action} ` : ` made ${elem.meta.count} ${elem.meta.action} to ` }
+        <a href={elem.meta.article_url}>{elem.meta.headline}</a>
+        <span>{` ${moment(elem.timestamp).from(moment())}`}</span>
+      </li>))
```

Comment:
re point # 2 -- one approach would be to float the icon left, put the text in an inline-block or block element, and slap a margin-left on it. it should wrap at the same indent:

![screen shot 2017-02-12 at 9 51 39 pm](https://cloud.githubusercontent.com/assets/887504/22872378/9f545eac-f16d-11e6-8298-ddf02b46a6d0.png)

```html
    <style>
        .a { margin-left: 2rem; background: #ddd; max-width: 25rem; margin-top: 1rem; }
        i { float: left; }
    </style>

    <div>
        <i class="fa fa-server"></i>
        <div class="a">It was a bright cold day in April, and the clocks were striking thirteen.</div>
    </div>
    <div>
        <i class="fa fa-feed"></i>
        <div class="a">Winston Smith, his chin nuzzled into his breast in an effort to escape the vile wind, slipped quickly through the glass doors of Victory Mansions, though not quickly enough to prevent a swirl of gritty dust from entering along with him.</div>
    </div>
```
(this is divs, not `li`s, but same principle)